src/sections/trade/sections/otc/modals/PlaceOrder.tsx

```diff
@@ -51,16 +51,29 @@ export const PlaceOrder = ({
     mode: "onChange",
   })
 
-  useEffect(() => {
-    form.trigger()
-  }, [form])
+  const { setError, clearErrors } = form
 
   const { api, assets } = useRpcProvider()
   const assetOutMeta = aOut ? assets.getAsset(aOut.toString()) : undefined
   const assetOutBalance = useTokenBalance(aOut, account?.address)
   const assetInMeta = aIn ? assets.getAsset(aIn.toString()) : undefined
   const assetInBalance = useTokenBalance(aIn, account?.address)
 
+  useEffect(() => {
+    form.trigger()
+  }, [form])
+
+  useEffect(() => {
```

Comment:
use can add that validation within amountIn form item

Reply:
Hi

Originally I had this like this but figured maybe the user could use this error message before inputing amounts for both assets. I'll revert back to being in the form controller itself.

Reply:
Resolved in next push.